### PR TITLE
Feature/r0008 impl

### PR DESCRIPTION
**Pull request by amitschendel**

### CODE REVIEW

File: pkg/engine/rule/r0008_malicious_ssh_connection.go [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+	return &R0008MaliciousSSHConnection{accessRelatedFiles: false,
+		sshInitiatorPid:           0,
+		configFileAccessTimeStamp: 0,
+		allowedPorts:              []uint16{22},
+	}
+}
+
+func (rule *R0008MaliciousSSHConnection) SetParameters(params map[string]interface{}) {
+	if params["allowedPorts"] != nil {
+		rule.allowedPorts = params["allowedPorts"].([]uint16)
```
Comment (Bezbran) [inline pkg/engine/rule/r0008_malicious_ssh_connection.go:89]:
```suggestion
		rule.allowedPorts, ok := params["allowedPorts"].([]uint16)
		if !ok{
		 // print error?
		}
```

File: pkg/engine/rule/r0008_malicious_ssh_connection.go [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+			return true
+		}
+	}
+	return false
+}
+
+func (rule *R0008MaliciousSSHConnection) Requirements() RuleRequirements {
+	return RuleRequirements{
+		EventTypes:             []tracing.EventType{tracing.OpenEventType, tracing.NetworkEventType},
+		NeedApplicationProfile: false,
```
Comment (Bezbran) [inline pkg/engine/rule/r0008_malicious_ssh_connection.go:161]: I think we should have a clear decision what should be in the app profile and what in the parameters.
This is a comment with no AI

File: pkg/engine/rule/r0008_malicious_ssh_connection.go (hunk 1 above)
Comment (Bezbran) [reply pkg/engine/rule/r0008_malicious_ssh_connection.go:89]: Alternatively - you can tweek the CRD to handle the input checks of the allowed parameters for you

**Review by Bezbran — CHANGES_REQUESTED** [review]
1. I didn't review the logic of the rule itself, just the envelope.
2. We must add all the files which dealing with all rules to the code which updates the README of the rules...

File: pkg/engine/rule/r0008_malicious_ssh_connection.go (hunk 1 above)
Comment (amitschendel) [reply pkg/engine/rule/r0008_malicious_ssh_connection.go]: done

File: pkg/engine/rule/r0008_malicious_ssh_connection.go (hunk 2 above)
Comment (amitschendel) [reply pkg/engine/rule/r0008_malicious_ssh_connection.go:163]: This rule doesn't need application profile because imo

**Review by slashben — APPROVED** [review]

**Review by Bezbran — APPROVED** [review]
In the future we better to explain in the error what we expcted to get and we actually got

**Pull request merged**